packages/decorators/src/subcommand-decorators.ts

```diff
@@ -25,7 +25,7 @@ export function ChatInputSubcommand(name: string): MethodDecorator {
 		const mappings: SubcommandMappingArray = Reflect.get(target, 'subcommandMappings');
 		const subcommand = mappings.find((mapping) => mapping.name === name);
 		if (subcommand) {
-			(subcommand as any).messageRun = propertyKey;
+			(subcommand as any).chatInputRun = propertyKey;
```

Comment:
So now you have the opposite problem - revert this to `messageRun` since this is a MessageSubcommand

Reply:
what